CMS-Enterprise/easi-app
# [EASI-4586] Add AI Question to System Intake and as Subject Area for TRB

# EASI-4586

## Description
- The ITGO and TRB teams have gotten requirements to start capturing Artificial Intelligence (AI) information as a part of the IT intake and TRB processes. This work adds:
  - New Yes/No question in the System Intake Request Details section that asks requesters if their project uses AI
  - New 'AI' Subject Area in TRB

## How to test this change
- Go through ITGO and TRB requester workflows to ensure that new question/subject area tag function correctly
- Ensure that fields are properly displayed in intake/TRB review as well as ITGO CSV export and Portfolio Update Report

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
